editor/components/block-settings-menu/index.js

```diff
@@ -25,7 +25,10 @@ function BlockSettingsMenu( {
 	uids,
 	onSelect,
 	focus,
-	renderBlockMenu = ( { children } ) => children }
+	renderBlockMenu = ( { children } ) => children,
+	draggable,
+	onDragStart,
+	onDragEnd }
```

Comment:
Minor: We should be consistent with opening brace and closing brace positioning relevant to the object members, i.e. this `}` should be on the following line.